README.md

```diff
@@ -1,2 +1,42 @@
-# Mighty-Seal-GO
-Go game in react
+## Description
+
+![](https://upload.wikimedia.org/wikipedia/commons/thumb/f/f3/Go-board-animated.gif/120px-Go-board-animated.gif)
+
+Two people can play [Go](https://en.wikipedia.org/wiki/Go_%28game%29) on the same webpage.
+
+[Go](https://en.wikipedia.org/wiki/Go_%28game%29) is an existing, fixed feature set.
+
+Instructions for playing the game: 
+Within the my-app directory, do npm i and npm start
+Game pieces can be placed by clicking on the game board
+Players can skip turn by pressing the pass button
+
+
```

Comment:
I would squash these first two commits together since they're both just changes to the readme.

You can also do git commit --amend to add more changes to the previous commit without making a new one.